esFill.js

```diff
@@ -0,0 +1,8 @@
+const exports = {
```

Comment:
Same as for other PR: what does this file do, and why is it included in the pull request?